database/migrations/2017_12_13_081248_create_post_tags_table.php

```diff
@@ -29,6 +29,6 @@ public function up()
     public function down()
     {
 
-        Schema::dropIfExists('post_tag');
+        Schema::dropIfExists('post_tags');
```

Comment:
sao tên table ở relationship lại như vậy anh? Phải đặt là "post_tag" mới đúng anh.
Anh tham khảo ở ttps://laravel.com/docs/5.5/eloquent-relationships#many-to-many